src/features/Banner/component.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+// import components
+import Navbar from '../../shared/Navbar';
+import SearchMovie from '../../shared/SearchMovie';
+import './style.scss';
+
+export default class Banner extends Component 
+{
+	constructor(props){
+		super(props);
+	}
+
+    render() 
+{
+        return (
+	<section className='bannerSection'>
+		<Navbar 
+			isHidden={this.props.isHidden} 
```

Comment:
to make lines like this a bit shorter, we usually do destructuration of the props before "return"
so it'll look like this in the end
```suggestion
			isHidden={isHidden} 
```

Reply:
This is added now